testutils/setup.go

```diff
@@ -159,8 +160,26 @@ type TestConfig struct {
 	UseS3Fallback      bool
 }
 
+// NewTestConfig returns a new TestConfig, which only enables the single eigenDA backend needed, based on backend
+// dispersal target
 func NewTestConfig(backend Backend, disperseToV2 bool) TestConfig {
+	var backendsToEnable []common.EigenDABackend
+	if disperseToV2 {
+		backendsToEnable = []common.EigenDABackend{common.V2EigenDABackend}
+	} else {
+		backendsToEnable = []common.EigenDABackend{common.V1EigenDABackend}
+	}
+	return NewTestConfigSpecifyingBackends(backend, disperseToV2, backendsToEnable)
+}
+
+// NewTestConfigSpecifyingBackends returns a new test config, which enables a specific set of eigenDA backends
+func NewTestConfigSpecifyingBackends(
+	backend Backend,
+	disperseToV2 bool,
+	backendsToEnable []common.EigenDABackend,
+) TestConfig {
```

Comment:
nit: I personally really don't like this kind of proliferation of constructors for every field. That's why I like the functional options pattern. Or add it to the main `NewTestConfig` constructor with an ellipsis `...` to make it optional.

Reply:
used variadic param in [e9cd6a5](https://github.com/Layr-Labs/eigenda-proxy/pull/348/commits/e9cd6a549bb475179a7c84813b9292a3c7699af9)